### PR TITLE
add staticDir-flag

### PR DESCRIPTION
**Pull request by led0nk**
> Since things are time critical
:sweat_smile: 

should solve this issue:
- #61 

`static-dir` has to be provided by:
`-static-dir=some/path/to/any/folder/static`

Caching is still a problem, because you have to clear browser cache to see updates.

Testing locally: :smile: 

image

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ func main() {
 		dbStr       = flag.String("db", "json://testdata", "database connection string")
 		otlpAddr    = flag.String("otlp-grpc", "", "default otlp/gRPC address, by default disabled. Example value: localhost:4317")
 		logLevelArg = flag.String("log-level", "INFO", "log level")
+		staticDir   = flag.String("static-dir", "./intern/server/static", "path to static directory")
```
Comment (frzifus) [inline cmd/server/main.go]:
```suggestion
		staticDir   = flag.String("static-dir", "", "path to static directory")
```

File: intern/server/server.go [hunk 2]
```diff
@@ -78,11 +77,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		username: password,
 	}))...)
 
-	staticDir, err := fs.Sub(staticFS, "static")
-	if err != nil {
-		panic(err)
-	}
-	mux.StaticFS("/static", http.FS(fs.FS(staticDir)))
```
Comment (frzifus) [inline intern/server/server.go]: Can we keep the embedded one and only use `staticDir` when its configured?

**Review by frzifus — COMMENTED** [review]
LGTM, one last thing :)

File: cmd/server/main.go (hunk 1 above)
Comment (led0nk) [reply cmd/server/main.go]: :heavy_check_mark:

File: intern/server/server.go (hunk 2 above)
Comment (led0nk) [reply intern/server/server.go]: done :)

**Review by frzifus — APPROVED** [review]
thanks @led0nk !!

**Pull request merged**